docs/ref/contrib/postgres/fields.txt

```diff
@@ -599,7 +599,7 @@ excluded; that is, ``[)``.
 
     Regardless of the bounds specified when saving the data, PostgreSQL always
     returns a range in a canonical form that includes the lower bound and
-    excludes the upper bound; that is ``[)``.
+    excludes the upper bound; According to `RangeBoundary() expressions`_, that is ``'[)'``. 
```

Comment:
IMO, we shouldn't mix fields descriptions with `RangeBoundary()` expression. `[)` is set according to the `PostgreSQL` implementation, not according to our expression. Maybe:

```
... that is ``[)``. The PostgreSQL documentation has more details about `bounds
<https://www.postgresql.org/docs/current/rangetypes.html#RANGETYPES-INCLUSIVITY>`_.
```

Reply:
I agree with you. This is not related to Django's RangeBoundary expression, so i'll fix it as you suggested.

Reply:
Btw, Is it intended end of Semicolon like this?
`excludes the upper bound; `

Reply:
It depends, in this case we can use a comma.